server/store/bids.js

```diff
@@ -91,3 +91,5 @@ module.exports = {
   getBid,
   deleteBidsForNeed,
 };
+
+const setBidTTL = async bidId => redis.expire(`bids:${bidId}`, config('bids_ttl'));
```

Comment:
As @TalAter mentioned in a similar pr, https://github.com/DAVFoundation/missioncontrol/pull/159#pullrequestreview-97387026

> I think you can remove the async from this line. We don't actually have to await for a response from this one.